src/style.css

```
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  width: 100vw;
+  padding-left: 50px;
+  padding-right: 50px;
```

```suggestion
  padding: 0 50px;
```
You can use shorthand here